state/runtime/precompiled/precompiled.go

```diff
@@ -77,7 +77,7 @@ func (p *Precompiled) setupContracts() {
 	p.register(contracts.BLSAggSigsVerificationPrecompile.String(), &blsAggSignsVerification{})
 
 	// Console precompile
-	p.register(contracts.ConsolePrecompile.String(), &console{})
+	// p.register(contracts.ConsolePrecompile.String(), &console{})
```

Comment:
Linter complains, because the block is ending with a comment. We need to trick it out somehow...

Reply:
Solved!